tests/src/assets/include/single-include-url.yaml

```diff
@@ -0,0 +1,6 @@
+---
+"1":
+  value: true
+include_0: https://raw.githubusercontent.com/rdkcentral/ut-control/feature/gh10-support-custom-tag-yaml/tests/src/assets/include/2s.yaml
```

Comment:
ok this won't exist after you commit it, because we'll delete the branch of course.

You can refer to it from `master` instead of `feature/gh10-support-custom-tag-yaml` before you check in, you should try a curl on the command that you change of course, to make sure you got the url